src/main/java/dk/kb/util/yaml/YAML.java

```diff
@@ -1370,4 +1424,8 @@ public String lookup(String key) {
             }
         }
     }
+
+    void accept(YAMLVisitor visitor) {
+        visitor.visit(this);
```

Comment:
This seems to be a strict implementation of the visitor pattern, so it should probably be Javadoc'ed with a mention of that. Alternatively the "object accepts visitor" could be switched to "visitor accepts object" (in line 612) which seems to align better with the concrete implementation.